manta-pay/Cargo.toml

```diff
@@ -78,7 +78,7 @@ simulation = [
 std = ["manta-accounting/std", "manta-util/std"]
 
 # Testing Frameworks
-test = ["manta-accounting/test", "manta-crypto/test"]
+test = ["manta-accounting/test", "manta-crypto/test", "manta-crypto/getrandom"]
```

Comment:
Instead of adding to the `test` feature, you should add this to the `dev-dependencies`. The `test` feature is for _exported_ test content rather than non-exported tests. NB: the `#[cfg(test)]` does not observe this feature only `#[cfg(feature = "test")]` does.

Reply:
done